LibraryWebApp/Controllers/ReservationController.cs

```diff
@@ -24,19 +23,69 @@ public async Task<IActionResult> Index()
         {
             var userName = HttpContext.User.Identity.Name;
             var user = await _userManager.FindByNameAsync(userName);
-            IEnumerable<Reservation> reservationBook =_db.Reservation.Include(x => x.ReservationBook).ThenInclude(x=>x.Book).Where(x=>x.IdClient == user.Id);
+            IEnumerable<Reservation> reservationBook = _db.Reservation.Include(x => x.ReservationBook).ThenInclude(x => x.Book).Where(x => x.IdClient == user.Id).Include(x=>x.Fee);
             return View(reservationBook);
         }
         public IActionResult Delete(int id)
         {
-            var obj = _db.Reservation.Find(id);
-            _db.Reservation.Remove(obj);
+            var reservation = _db.Reservation.Find(id);
+            _db.Reservation.Remove(reservation);
             foreach (var reservationBook in _db.ReservationBook.Where(x => x.IdReservation == id))
             {
                 _db.ReservationBook.Remove(reservationBook);
             }
             _db.SaveChanges();
-            return RedirectToAction(nameof(Index));
+            if (User.IsInRole(WC.AdminRole))
+            {
+                return RedirectToAction(nameof(IndexAdmin));
+            }
+            else
+            {
+                return RedirectToAction(nameof(Index));
+            }
+        }
+        public IActionResult IndexAdmin()
+        {
+            IEnumerable<Client> reservations = _db.Client.Include(x => x.Reservation).ThenInclude(x=>x.ReservationBook).ThenInclude(x=>x.Book).Where(x=>x.Reservation.Count>0).Include(x=>x.Reservation).ThenInclude(x=>x.Fee);
```

Comment:
Spacing and break the line
It's not list of reservation, but it's list of clients?